utkarsh component/responsive image/img.css

```diff
@@ -0,0 +1,587 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+Thai+Looped:wght@300&display=swap');
+
+body{
+    font-family: 'IBM Plex Sans Thai Looped', sans-serif;
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+background-color:  #31353a;
+height: 100%;
+
+}
+
+* {
+   box-sizing: border-box;
+--primary-color: #ecfeff ;
+--secondary-color: #4908fa;
+}
+
+
+#sidebarMenu{
+    height: 200%;
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 120px;

```

Comment:
please do not use absolute unit that is pixel, try using rems for better responsiveness